### ISSUE TITLE
Helper method

### ISSUE THREAD

**Issue by gitmajo** [OriaTori/CS_Database#17]
It seems to me that this method is used as helper, maybe it should be placed in private section. 
Please check mi @LordLukin. 

https://github.com/OriaTori/CS_Database/blob/483302198cbc63f8a8550a2fb4cff16d29839f91/include/Database.hpp#L26

**Comment by LordLukin** [comment]
Yes, it's a helper function, which is used only in this class, so it should be private.
The public interface is defined by requirements.
